### PR TITLE
map form type

### PR DESCRIPTION
**Pull request by JarekW**

### CODE REVIEW

**Comment by JarekW** [comment]
@fsi-open

File: Resources/views/Form/form_ckeditor_layout.html.twig [hunk 1]
```diff
@@ -20,4 +20,28 @@
     CKEDITOR.replace('{{ form.vars.id }}', options);
 </script>
 {% endspaceless %}
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+
+{% block fsi_map_widget %}
```
Comment (chives) [inline Resources/views/Form/form_ckeditor_layout.html.twig]: I'd move this block to separate template and add separate compiler pass to register it as a form resource

**Comment by JarekW** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

**Comment by chives** [comment]
@JarekW please update the docs

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
@fsi-open

**Comment by JarekW** [comment]
todo: behat test that checks for map is loaded

**Comment by JarekW** [comment]
@fsi-open rebased

**Comment by JarekW** [comment]
@fsi-open

**Comment by JarekW** [comment]
@fsi-open ?

**Comment by chives** [comment]
:+1:

File: Resources/public/js/map.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+                map,
+                marker;
+
+            if (!latitudeField.length || !longitudeField.length || !mapWrapper.length) {
+                return;
+            }
+
+            if (latitudeField.val() && longitudeField.val()) {
+                location = {
+                    lat: parseFloat(latitudeField.val().replace(',','.')),
```
Comment (rn0) [inline Resources/public/js/map.js]: i would extract `parseFloat(value.replace(',','.'))` as 

```js
function normalizeNumber(value)
{
     return parseFloat(value.val().replace(',','.'))
}
```

**Comment by szymach** [comment]
:+1:

File: Resources/public/js/map.js (hunk 2 above)
Comment (JarekW) [inline Resources/public/js/map.js]: changed :)

**Comment by rn0** [comment]
:+1:

**Pull request merged**